geode-redis/README.md

```diff
@@ -159,54 +159,54 @@ start server \
-|                    	| SUNIONSTORE                                        	| CLUSTER RESET                  	|
-|                    	| TIME                                               	| CLUSTER SAVECONFIG                |
-|                    	| UNLINK [1]                                         	| CLUSTER SET-CONFIG-EPOCH          |
+| DECR               	| BITPOS                                             	| ACL GENPASS                   	|
+| DEL                	| DBSIZE                                             	| ACL GETUSER                   	|
+| EXISTS             	| DECRBY                                                | ACL HELP                      	|
+| EXPIRE             	| ECHO                                               	| ACL LIST                      	|
+| EXPIREAT           	| FLUSHALL                                           	| ACL LOAD                      	|
+| GET                 	| FLUSHDB                                            	| ACL LOG                       	|
+| GETRANGE              | GETBIT                                             	| ACL SAVE                      	|
+| HDEL        	     	| GETSET                                                | ACL SETUSER                   	|
+| HEXISTS           	| HINCRBY                                            	| ACL USERS                     	|
+| HKEYS                	| HSCAN                                                 | ACL WHOAMI                    	|
+| HGET               	| INCR                                                  | BGREWRITEAOF                  	|
+| HGETALL              	| INFO                                                  | BGSAVE                        	|
+| HINCRBY               | MSET                                                  | BITFIELD   
+| HINCRBYFLOAT       	| MSETNX                                                | BLPOP                          	|
+| HMGET             	| PSETEX                                             	| BRPOP                                                                	|
+| HMSET                 | SCAN                                                  | BRPOPLPUSH                    	| 	                     	|
+| HSCAN                 | SCARD                                                 | BZPOPMAX                      	|
+| HSET                  | SDIFF                                                 | BZPOPMIN                      	|
+| HSETNX                | SDIFFSTORE                                            | CLIENT CACHING                	|
+| HSTRLEN               | SELECT                                                | CLIENT GETNAME                	|
+| HLEN                  | SETBIT                                                | CLIENT GETREDIR               	|
+| HVALS                 | SETEX                                                	| CLIENT ID                     
```

Comment:
Could we put these hash commands in alphabetical order? (looks like just `HKEYS` and `HLEN` need to be moved.)

Reply:
sure.  done.